### ISSUE TITLE
Why should I fork this?

### ISSUE THREAD

**Issue by Bloggerschmidt** [hamukazu/lets-get-arrested#38]
The girl was arrested for "putting a link" and not writing the code. It's all about sharing. You only need one site with this alert JavaScript. Multiple persons can share one site. No need for multiple sites here.

**Comment by pkmnfrk** [comment]
Because then there is only one criminal, which is @hamukazu. If everyone forks this, then everyone is guilty!

**Comment by hamukazu** [comment]
Yes. The girl just "put a link." Thus just putting a link to the infinite loop of alert is regarded as a crime. However I strongly recommend to fork this project to be more felonious.
